cypress/fixtures/schedule.json

```diff
@@ -8,6 +8,7 @@
   "updated_at": 1685037152,
   "updated_by": "CookieCat",
   "scheduled_at": 0,
+  "branch": "master",
```

Comment:
not a comment for you but we should really change all these master references to main